src/common/mailing.ts

```diff
@@ -248,39 +248,39 @@ export const syncSubscriptionsWithActiveState = async ({
       }
 
       const data = users.map(({ id }) => ({
-        action: 'destroy',
+        action: 'delete',
```

Comment:
Just double check you don't call action elsewhere wrongly.

Reply:
So far we only implemented two areas, one for identify and one for delete. Should be as what is intended 🚀